code/_onclick/hud/screen_objects.dm

```diff
@@ -54,7 +63,198 @@
 
 /obj/screen/grab/attackby()
 	return
+/obj/screen/act_intent
+	name = "intent"
+	icon_state = "help"
+
+/obj/screen/act_intent/Click(location, control, params)
+	usr.a_intent_change("right")
+	if("help")
```

Comment:
Uh.. this is just checking if the string "help" exists... which since it's written in the if(), probably always does...

Also why do you guys still use the intent menu that rotates it to the right, instead of being able to click which intent you want?